tests/lora/utils.py

```diff
@@ -85,8 +85,13 @@ class PeftLoraLoaderMixinTests:
     unet_kwargs = None
     transformer_cls = None
     transformer_kwargs = None
+    vae_cls = AutoencoderKL
     vae_kwargs = None
 
+    text_encoder_target_modules = ["q_proj", "k_proj", "v_proj", "out_proj"]
+
+    output_identifier_attribute = "images"
```

Comment:
`.images` vs `.frames` in pipeline output. We could, I think, just access with `pipe(...)[0]` without using attribute names

Reply:
Changed this to `pipe(...)[0]` as it's cleaner